src/backend/direct.rs

```diff
@@ -344,15 +344,14 @@ impl crate::Context for Context {
         &self,
         options: &crate::RequestAdapterOptions<'_>,
         unsafe_extensions: wgt::UnsafeExtensions,
-        backends: wgt::BackendBit,
     ) -> Self::RequestAdapterFuture {
         let id = self.pick_adapter(
             &wgc::instance::RequestAdapterOptions {
                 power_preference: options.power_preference,
                 compatible_surface: options.compatible_surface.map(|surface| surface.id),
             },
             unsafe_extensions,
-            wgc::instance::AdapterInputs::Mask(backends, |_| PhantomData),
+            wgc::instance::AdapterInputs::Mask(wgt::BackendBit::all(), |_| PhantomData),
```

Comment:
we could get the mask from the instance, could we?

Reply:
Well technically we can only create a mask for the backends that sucessfully instantiated (unless we explicitly store backends in the `Instance` struct). And the way `pick_adapter` works means it would make no difference in any way, so...

Reply:
yep, hence we are all merged :)